### PR TITLE
feat: Move recovery phrase into 'settings'

### PR DESCRIPTION
**Pull request by Samuel3Shin**
### Description

This is an A/B test.

Control: current experience w/ recovery phrase only in the navigation menu

Treatment:
If the Firebase remote config `shouldShowRecoveryPhraseInSettings` is true, the user sees the recovery phrase in settings, not in the navigation menu.
However, if the user hasn't backed up his recovery phrase yet, then the user can see it in the navigation menu as well even though the remote config is true.

Screen Shot 2022-08-04 at 5 33 19 PM Screen Shot 2022-08-04 at 5 33 08 PM Screen Shot 2022-08-04 at 5 37 43 PM

### Other changes
N/A

### Tested
With iOS emulator

### Related issues
- https://linear.app/valora/issue/RET-302/move-recovery-phrase-into-settings

### Backwards compatibility
Yes

### CODE REVIEW

File: src/account/Settings.tsx [hunk 1]
```diff
@@ -92,6 +95,7 @@ interface StateProps {
   walletConnectEnabled: boolean
   biometryEnabled: boolean
   supportedBiometryType: BIOMETRY_TYPE | null
+  shouldShowRecoveryPhraseInSetting: boolean
```
Comment (kathaypacific) [inline src/account/Settings.tsx:98]: could we rename this to `shouldShowRecoveryPhraseInSettings`? 🙏🏻

**Review by kathaypacific — APPROVED** [review]
looks great!! 🚀

File: src/account/Settings.tsx (hunk 1 above)
Comment (Samuel3Shin) [reply src/account/Settings.tsx]: Yes, I just did it. Thanks!

**Comment by ValoraQA** [comment]
Hey @MuckT we have verified the above task on Latest iOS Test Flight Build V 1.39.1 and updated status in Test Rails

**Observation :** 
- If user has created account and not set up the recovery phrase , the recovery Phrase is seen in Hamburger menu
- If user has created account , set up the recovery phrase then recovery phrase is not seen in settings option , it is still seen in Hamburger menu
- If user has restored account ,then recovery phrase is not seen in settings option , it is still seen in Hamburger menu
- Will "Recovery Phrase" section moved to settings once the recovery phrase is setup or it is related to config ?

**Test rail links :**
**iOS 14:** https://valoraapp.testrail.io/index.php?/tests/view/61310

**Devices :** iPhone 12(14.7.1)

Cc: @Samuel3Shin 

Thanks.!

**Comment by ValoraQA** [comment]
Hey @MuckT we have verified the above task on Latest iOS Test Flight Build V 1.39.1 and updated status in Test Rails

**Observations**:

- If user has created account and not set up the recovery phrase , the recovery Phrase is seen in Hamburger menu
- If user has created account , set up the recovery phrase then recovery phrase is not seen in settings option , it is still seen in Hamburger menu
- If user has restored account ,then recovery phrase is not seen in settings option , it is still seen in Hamburger menu
- Will "Recovery Phrase" section moved to settings once the recovery phrase is setup or it is related to config ?

**Test rail links :**
**iOS 15:** https://valoraapp.testrail.io/index.php?/tests/view/61836

**Device** : iPhone 13 mini(15.1.1)

**Cc**: @Samuel3Shin

Thanks.!!

**Pull request merged**